server/src/main/java/io/crate/execution/engine/indexing/GroupRowsByShard.java

```diff
@@ -105,13 +107,16 @@ public GroupRowsByShard(ClusterService clusterService,
              UpsertResultContext.forRowCount());
     }
 
+    /**
+     * BiConsumer is needed for compatibility of the grouper with BatchIterators.partition
+     */
     @Override
     public void accept(ShardedRequests<TReq, TItem> shardedRequests, Row row) {
-        apply(shardedRequests, row);
+        apply(shardedRequests, row, false);
     }
 
     @Override
-    public TItem apply(ShardedRequests<TReq, TItem> shardedRequests, Row row) {
+    public TItem apply(ShardedRequests<TReq, TItem> shardedRequests, Row row, Boolean propagateError) {
         // `Row` can be a `InputRow` which may be backed by expressions which have expensive `.value()` implementations
```

Comment:
By default false (insert into swallows errors) but once we introduce a setting in https://github.com/crate/crate/issues/12218, we can pass that setting value here